<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCustomCommandDialog</class>
 <widget class="QDialog" name="AddCustomCommandDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddCustomCommandDialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="7" column="1" colspan="3">
      <widget class="QComboBox" name="buttonSelector"/>
     </item>
     <item row="2" column="1" rowspan="2" colspan="5">
      <widget class="QListView" name="buttonSequenceList"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Buton Sequence:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QPushButton" name="addButtonToSequenceButton">
       <property name="text">
        <string>Add Button To Sequence</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Command Name: </string>
       </property>
      </widget>
     </item>
     <item row="8" column="4">
      <widget class="QPushButton" name="addButtonToSequenceWithDS">
       <property name="text">
        <string>Add Button with DualShock</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Action Type: </string>
       </property>
      </widget>
     </item>
     <item row="9" column="5">
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="9" column="6">
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QComboBox" name="actionTypeSelector"/>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="commandNameField"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>AddCustomCommandDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
